games-engines/derclou/derclou-1.0.0~git.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="A burglary simulation game"
+DESCRIPTION="You play Matt Stuvysant, a man of Polish descent and criminal persuasion. \
```

Comment:
The first line of DESCRIPTION exceeds the 80 char recommended limit.
For your convenience, here are the first lines with the "right" line breaks:
```
DESCRIPTION="You play Matt Stuvysant, a man of Polish descent and criminal \
persuasion. You arrive in London with nothing but the clothes on your back and \
a desire to become the greatest criminal mind this century. Once getting a \
room and meeting some accomplices, you must begin your career. Starting with \
small shops and gradually working your way up to the theft of the Crown Jewels.
```